### DISCUSSION TITLE
Is the setting "Always show key overlay" necessary?

### DISCUSSION THREAD

**Discussion by nanashi-1** [ppy/osu#24491] [Q&A] [answered]
Now that the Key Counter is skinnable, it can just be remove in the skin editor if not needed. This makes this setting obsolete, or am I missing something?

![Screenshot from 2023-08-10 15-23-25](https://github.com/ppy/osu/assets/54083498/5dfad6e3-3538-47b9-a114-a068a9aabd3b)

**Comment by peppy** [comment, marked as answer]
It may be replaced in the future if we add a skinnable layer for "replay viewer". I can see this happening.

Right now it's still useful for the above case.
